src/main/resources/templates/demoAdmin.html

```diff
@@ -264,7 +264,7 @@
                                     </div>
                                 </div>
                                 <div id="client_connection">
-                                    <div ng-show="options.mqType === 'ACTIVE_MQ'">
+                                    <div ng-if="options.mqType === 'ACTIVE_MQ'">
                                         <div class="form-group">
```

Comment:
if you use ng-if, you can set two input fields have same name in different mode. So you don't need modify js file.